Makefile

```diff
@@ -15,7 +15,7 @@ D_FILE ?= "$(shell if [ "$(OS)" = Windows_NT ]; then echo "Dockerfile.wsc"; else
 DC_FILE ?= ".docker/docker-compose.yml" # Single docker-compose for Ubuntu/WSC, should be kept in sync with ENV_IMAGE_TAG.
 ENV_IMAGE_TAG="env_neo_go_image"
 
-REPO ?= "$(shell go list -m)"
+REPO ?= "$(shell go list -m | head -n 1)"
```

Comment:
That's something different.

Reply:
With `go.work` this will list all go mods and the first one is neo-go mod.